src/Card/Card.js

```diff
@@ -10,14 +10,36 @@ class Card extends Component {
   render(props) {
     return (
       <article className="card--container">
-        <img
-          className="card--image"
-          src={this.props.userCard.imageSource}
-          alt={this.props.userCard.cardName}
-        />
-        <div className="counter--container">
-          <h1 className="card--counter">{this.props.userCard.cardCount}x</h1>
-        </div>
+        {this.props.cardAreaView === "myCardList" && (
+          <div>
+            <img
+              className="card--image"
+              src={this.props.userCard.imageSource}
+              alt={this.props.userCard.cardName}
+            />
+            <div className="counter--container">
+              <h1 className="card--counter">
+                {this.props.userCard.cardCount}x
+              </h1>
+            </div>
+          </div>
+        )}
+
+        {this.props.cardAreaView === "expandedDeck" && (
+          <div>
+            <img
+              className="card--image"
+              src={this.props.cardImage}
+              alt={this.props.key}
+            />
+            <div className="counter--container">
+              <p>{this.props.price}</p>
+              {/* <h1 className="card--counter">
+                {this.props.userCard.cardCount}x
+              </h1> */}
+            </div>
+          </div>
+        )}
```

Comment:
Nice use of short circuit && operator for conditional rendering.

Reply:
Is there a way to display the number of cards the user has and the number of cards the deck has? Like as a fraction maybe? That might be hard to do, but I thinking that if the user has 2 of a card and the deck.cardCounts says the deck has 4 of that card, we could display `2 / 4` along the bottom of the card image.

And if we get really ambitious, maybe `4 / 4` could be colored green, `0 / 4` could be colored red, and numbers in between could be colored yellow.